slippers/templatetags/slippers.py

```diff
@@ -210,6 +210,8 @@ def render(self, context):
 def do_attrs(parser, token):
     tag_name, *attrs = token.split_contents()
 
+    attrs = set(attrs).union(set(settings.SLIPPERS_CORE_COMPONENT_ATTRS.split(" ")))
```

Comment:
```suggestion
    attrs = set(attrs) | set(settings.SLIPPERS_CORE_COMPONENT_ATTRS.split(" "))
```
Can also be written like this.

Reply:
https://github.com/QuickRelease/slippers/pull/23/files#diff-162ca2eb467e222e94284aeb8fe13591aad4f14357fb5b154e0d63c13c512223R213